scripts/cert.sh

```diff
@@ -13,7 +13,7 @@ openssl genrsa -out admin.key 2048
 openssl req -new -key admin.key -out admin.csr -subj '/C=US/ST=Washington/L=Seattle/O=Chef Software Inc/CN=chefadmin'
  
 #Use the CSR to generate the signed node Certificate:
-openssl x509 -extfile <(printf "subjectAltName=DNS:chefadmin") -req -in admin.csr -CA MyRootCA.pem -CAkey MyRootCA.key -CAcreateserial -out admin.pem -sha256
+openssl x509 -extfile <(printf "subjectAltName=DNS:chefadmin") -req -in admin.csr -CA MyRootCA.pem -CAkey MyRootCA.key -CAcreateserial -out admin.pem -sha256 -days 2042
```

Comment:
```suggestion
openssl x509 -extfile <(printf "subjectAltName=DNS:chefadmin") -req -in admin.csr -CA MyRootCA.pem -CAkey MyRootCA.key -CAcreateserial -out admin.pem -sha256 -days 3660
```
10years *366 days = 3660 Days in 10 years,  we can use this to generate certificate

Reply:
Updated